format_spec/FORMAT_SPEC.md

```diff
@@ -2,7 +2,7 @@
 
 
 :information_source: **Notes:**  
-- The current TileDB format version number is **5** (`uint32_t`).
+- The current TileDB format version number is **7** (`uint32_t`).
```

Comment:
What happened to **6**? Had we bumped the version by mistake in `dev`, or did we forget to add any format spec changes? Just a note here so that we follow up.

Reply:
Yep, that's what happened. The format version in `constants.cc` was bumped from 5 to 6 with fill values. This bumps both that and this to 7.